Example/Tests/Utilities/Throttler/ThrottlerTests.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ThrottlerTests.swift
+//  PovioKit_Tests
+//
+//  Created by Klemen Zagar on 05/12/2019.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import PovioKit
+
+class ThrottlerTests: XCTestCase {
+  
+  func testShouldExecuteWhenDelayed() {
+    let delay = 100
+    let waiting = delay + 10
+    let sut = Throttler(delay: .milliseconds(delay))
+    let expectation = self.expectation(description: "Executed")
+    var output: [String] = []
+    sut.execute {
+      output.append("A")
+    }
+    DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(waiting)) {
```

Comment:
Using dispatch for the wait is not a good practice. We should mock Throttler instead.

Reply:
Great. I'll improve it.